apps/courses/src/routes/index.tsx

```diff
@@ -24,54 +29,192 @@ const UnauthRoute = (props) => {
+  children,
+}) => (
+  <>
+    {!noHeader && <Header noScrolling={blackHeader} />}
+    <Box
+      minHeight="100vh"
+      display="grid"
+      gridTemplateRows={!inConcept ? '1fr' : '1fr auto'}
+    >
+      {children}
+      {!noFooter && <Footer />}
+    </Box>
+  </>
+);
+
 export default () => (
   <Routes>
-    <Route path="/" element={<Homepage />} />
-    <UnauthRoute path="signup" element={<Signup />} />
-    <UnauthRoute path="signin" element={<Signin />} />
+    <Route
+      path="/"
+      element={
+        <RouteWrapper>
+          <Homepage />
+        </RouteWrapper>
+      }
+    />
+    <UnauthRoute
+      path="signup"
+      element={
+        <RouteWrapper>
+          <Signup />
+        </RouteWrapper>
+      }
+    />
+    <UnauthRoute
+      path="signin"
+      element={
+        <RouteWrapper>
+          <Signin />
+        </RouteWrapper>
+      }
+    />
     <Route path="users">
       <Route path="/" element={<Navigate to="/" />} />
-      <AuthRoute path="dashboard" element={<Dashboard />} />
-      <AuthRoute path="settings" element={<Settings />} />
-      <Route path=":uid" element={<Profile />} />
+      <AuthRoute
+        path="dashboard"
+        element={
+          <RouteWrapper>
+            <Dashboard />
+          </RouteWrapper>
+        }
+      />
+      <AuthRoute
+        path="settings"
+        element={
+          <RouteWrapper>
+            <Settings />
+          </RouteWrapper>
+        }
+      />
+      <Route
+        path=":uid"
+        element={
+          <RouteWrapper>
+            <Profile />
+          </RouteWrapper>
+        }
+      />
     </Route>
     <Route path="courses">
-      <Route path="/" element={<CoursePage which="search" />} />
+      <Route
+        path="/"
+        element={
+          <RouteWrapper>
+            <CoursePage which="search" />
+          </RouteWrapper>
+        }
+      />
       <Route path=":course">
-        <Route path="/" element={<CoursePage which="overview" />} />
+        <Route
+          path="/"
+          element={
+            <RouteWrapper>
+              <CoursePage which="overview" />
+            </RouteWrapper>
+          }
+        />
         <AuthRoute
           path="complete"
-          element={<CoursePage which="courseComplete" />}
+          element={
+            <RouteWrapper blackHeader>
+              <CoursePage which="courseComplete" />
+            </RouteWrapper>
+          }
         />
         <AuthRoute path="project">
-          <AuthRoute path="/" element={<CoursePage which="project" />} />
+          <AuthRoute
+            path="/"
+            element={
+              <RouteWrapper noHeader>
+                <CoursePage which="project" />
+              </RouteWrapper>
+            }
+          />
           <AuthRoute
             path="complete"
-            element={<CoursePage which="projectComplete" />}
+            element={
+              <RouteWrapper blackHeader noFooter>
```

Comment:
There should be `noHeader` here, and not `blackHeader`. The black header you're seeing in the designs is the course `<Header>` component, which is always black. The only page that has the main `<Header>` component as black is the course complete page.